app/components/Poll/Poll.css

```diff
@@ -1,185 +1,167 @@
-@import '~app/styles/variables.css';
-
-.optionWrapper {
-  justify-content: center;
-  min-height: 120px;
-  cursor: pointer;
-  position: relative;
-}
-
-.pollTable {
+.poll {
+  margin: 0 auto;
+  background-color: var(--lego-card-color);
   width: 100%;
-  font-size: 14px;
-}
-
-.pollTable td {
-  border: 0;
-  padding: 5px;
+  min-width: 250px;
+  max-width: 500px;
+  overflow-y: hidden;
+  border-radius: 15px;
```

Comment:
I'd prefer if the `border-radius` would match the rest of the front page, as it would make it stick out less. Or do you think that looks ugly?